css/style.css

```diff
@@ -0,0 +1,16 @@
+@media only screen and (max-width: 600px) {
+  body {
+    background-color: lightblue;
+  }
+}
+
+#welcome-section {
+  height: 100vh;
+}
+
```

Comment:
- [ ] I think it will look better if you remove spaces between codes. Space between lines of code makes the code look unprofessional.

Reply:
- [ ] Great job giving this suggestion. However, the tone could be improved to make it more constructive and positive like this one:

  "Great work on the code! Just a small suggestion: removing the spaces between the lines of code could make it look even more professional. Keep up the good work! 🙌"

  _**Use this as an example and tailor it to your own feedback.**_ 🙌

---- 
  #### Remember that you can edit your review comments here:

  ![image](https://user-images.githubusercontent.com/107952724/235543841-6988b256-c4cb-4d52-b4d3-fd54c6d2adfc.png)